### DISCUSSION TITLE
How to Limit Search Scope to a Specific Site in PnP Modern Search

### DISCUSSION THREAD

**Discussion by tatsutanaka** [microsoft-search/pnp-modern-search#3992] [Q&A]
I have a question regarding the PnP Modern Search Web Parts.

If I want to limit the search scope to content within a specified site only, how should I configure the web part settings? I have entered "{Site.id._guid}" in the Query template of "PnP - Search Results," but this includes sub-sites of the specified site in the search scope (since the parent site and sub-sites share the same site ID). Is there a unique ID or method to specify only the parent site?

**Comment by wobba** [comment]
If you want to dynamic limit results to your current site you can use this in the query template:

`{searchTerms} Path:{Site.absoluteUrl}` or `{searchTerms} SiteID:{Site.id._guid}`

If you want to limit to a site, not sitecollection, use `WebID:{Web.id._guid}`

and in 2024 you should not really use sub-sites any longer - just saying 😉 That should have stopped over a decade ago at least.
